omod/src/main/java/org/openmrs/module/registrationapp/fragment/controller/field/DropDownFragmentController.java

```diff
@@ -19,14 +19,13 @@ public void controller(FragmentConfiguration config) throws Exception {
         if (config.getAttribute("conceptSet") != null) {
             List < Map < String, Object >> options = (List<Map<String, Object>>) config.getAttribute("options");
             if (options == null) {
-            	options = new ArrayList < Map < String, Object >> ();
+            	options = new ArrayList <Map<String, Object>> ();
```

Comment:
```suggestion
            	options = new ArrayList <Map<String, Object>> ();
            	options = new ArrayList<> ();
```
Using Diamond operators this can also be refactored